src/rules/unit-blacklist/index.js

```diff
@@ -28,10 +27,17 @@ export default function (blacklistInput) {
       const { value } = decl
 
       valueParser(value).walk(function (node) {
-        if (node.type === "function" && node.value.toLowerCase() === "url") { return false }
-        if (node.type !== "word" || isVariable(node.value)) { return }
+        // Remove interpolation characters
+        const value = node.value.replace(/[#@{}]+/g, "")
```

Comment:
@davidtheclark sorry, i don't right column count get from `node` and `valueParser` use `value` with interpolation. All ok.